iohk-monitoring/src/Cardano/BM/Backend/Switchboard.lhs

```diff
@@ -249,7 +246,7 @@ instance (FromJSON a, ToObject a) => IsBackend Switchboard a where
 
                 Async.async $ qProc countersMVar
 
-        q <- atomically $ TBQ.newTBQueue 2048
+        q <- atomically $ TBQ.newTBQueue 1000000
```

Comment:
we do not want to have such a big queue in production. chould be conditional on a flag?

Reply:
Done.